sandbox/settings.py

```diff
@@ -296,6 +296,9 @@ class Base(DRFMixin, RichieCoursesConfigurationMixin, Configuration):
     RICHIE_ES_HOST = values.Value(
         "elasticsearch", environ_name="RICHIE_ES_HOST", environ_prefix=None
     )
+    RICHIE_ES_INDICES_PREFIX = values.Value(
+        "richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
```

Comment:
You need to set a default of `None` if you want the code in `src/richie/apps/search/defaults.py ` to work:
```suggestion
        "richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
```

Maybe add a test to validate precedence on index name?

Reply:
I'm not sure how to test this since it's something that runs at bootstrap time. Also your suggested change seems not to be working but I see what you wanted to suggest 👍